### ISSUE TITLE
Retrieve position when app is closed

### ISSUE THREAD

**Issue by sayed-seliman** [jamesmontemagno/GeolocatorPlugin#297]
Hi,

is it possible to retrieve position, on Android and iOS, also when the app is closed?

Thanks

**Comment by rohitvipin** [comment]
You can use a service in android - https://github.com/Azure-Samples/MyDriving/tree/master/src/MobileApps/MyDriving

**Comment by chrisfoulds** [comment]
yes service on droid, background task on iOS -both work perfectly

**Comment by Yahya0007** [comment]
Even Android 8.0 and on wards?
